### PR TITLE
[simpleini cpp-redis tacopie] ports update

### PR DESCRIPTION
**Pull request by PhoebeHui**
Related issue https://github.com/microsoft/vcpkg/issues/8240

1. simpleini: fix the case issue
2. cpp-redis and tacopie: update the REPO

### CODE REVIEW

File: ports/simpleini/portfile.cmake [hunk 1]
```diff
@@ -9,6 +9,6 @@ vcpkg_from_github(
     HEAD_REF master
 )
 
-file(COPY ${SOURCE_PATH}/simpleini.h DESTINATION ${CURRENT_PACKAGES_DIR}/include)
+file(COPY ${SOURCE_PATH}/Simpleini.h DESTINATION ${CURRENT_PACKAGES_DIR}/include)
```
Comment (JackBoosY) [inline ports/simpleini/portfile.cmake]: What has been changed here?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/simpleini/portfile.cmake (hunk 1 above)
Comment (PhoebeHui) [reply ports/simpleini/portfile.cmake]: Change simpleini.h to Simpleini.h, Linux is case sensitive.

**Review by JackBoosY — APPROVED** [review]

File: ports/simpleini/portfile.cmake (hunk 1 above)
Comment (whidbey) [inline ports/simpleini/portfile.cmake]: I think it should be SimpleIni.h not Simpleini.h  sorry for the later comment

**Review by whidbey — COMMENTED** [review]
sorry later reply but Simpleini.h should be SimpleIni.h however

**Pull request merged**